Sources/AblyChat/RoomReactions.swift

```diff
@@ -2,10 +2,18 @@ import Ably
 
 public protocol RoomReactions: AnyObject, Sendable, EmitsDiscontinuities {
     func send(params: RoomReactionParams) async throws
```

Comment:
I think this method should either return a `Reaction` or `mockAsyncSequence` should be able to emit events, wdyt @lawrence-forooghian

Reply:
Why do you think it should return an `Reaction`?

And what do you mean by "`mockAsyncSequence` should be able to emit events"? Do you mean that there should be a way to make the `mockAsyncSequence` that you pass to `Subscription` emit an element? If so, there is — given that `mockAsyncSequence` can be any arbitrary `AsyncSequence`, you just need to write an `AsyncSequence` implementation that provides that API.

Reply:
https://github.com/ably-labs/ably-chat-swift/commit/b4d7de03074f3f737d8f578d85b34e3d4aacb268